test/supertest.js

```diff
@@ -150,7 +154,7 @@ describe('request(app)', function(){
 
     request(app)
     .get('/')
-    .expect(302, done);
+    .expect(302, function() { done(); });
```

Comment:
It is because `superagent` 1.x now sends any HTTP error (anything that is not a 2XX response code) to the callback as the first argument.  It works like this:

```
request(app)
  .get('/')
  .expect(302, function (error) {
    (error !== null).should.be.true;
    done();
  });
```

Happy to make that change to make the test more understandable.